Triaging-process.md

```diff
@@ -0,0 +1,73 @@
+
+### Steps to triage incoming issues for each team (second-stage triage):
+Team leads should do an audit every week to categorise new issues in their GitHub Projects board, following these steps:
+
+1. Verify that the issue lies within the scope of the team (see definitions below).
+    - If not, assign the issue to the appropriate project/team.
+2. Determine whether the issue is a bug or a enhancement
+    - If the issue is a feature request (enhancement), contact the relevant PM (product manager) for the team and ask them to take a look and decide whether it needs to be worked upon. (See the list of PMs [below](#pm-contacts-for-each-team).)
+3. (For simple UI issues) Make sure the bug is reproducible
+    - The issue should be consistently reproducible on either develop or the test/backup server.
+    - If that isn’t the case, ask the issue author for clarity.
+4. Once the issue has been confirmed, the team lead should add the  appropriate ‘Impact’ label (asking the PM or tech lead for help if they’re not sure), as well as the appropriate ‘Work’ label to the issue.
+    - ‘Impact’ defines how much the issue affects the functionality of the user-facing features. There are 3 associated labels: **Impact: High**, **Impact: Medium** and **Impact: Low**. For example, the issue is considered to have a high impact if it breaks a critical feature or blocks a release. General bugs (non-blocking) and improvements can be considered to be of medium impact while issues that occur in a few edge cases or affect a small number of users are issues with a low impact.
+
+        - Severity:
+            - **Severe**: Blocks or represents a primary user journey – e.g. end user playing a lesson, developer installing Oppia.
+            - **Moderate**: blocks or represents a secondary user journey – e.g. end user changing a profile picture, developer easily seeing the output of frontend tests at the bottom of the CI log.
+            - **Minor**: Neither of the others, relates to “polish”.
+
+        <br />
+
+        
+        |     | Severity       | Severe | Moderate | Minor |
+        | --- | -------------- | ------ | -------- | ----- |
+        | **# users affected** |        |          |       |
+        | (roughly) 60+%      || High   | Medium   | Low   |
+        | (roughly) 5-60%     || High   | Low      | Low   |
+        | (roughly) < 5%      || Medium | Low      | Low   |
+
+        Table: How “impact” is determined.
+
+        <br />
+
+    - ‘Work’ defines how hard it would be to solve/resolve a particular issue. This is generally correlated with the amount of ambiguity in the task. We use the following rough system for assigning this label:
+        - Solution is clear and broken into good-first-issue-sized chunks – assign **Work: Low**
+        - Otherwise, if the means to find the solution is clear – assign **Work: Medium**
+        - Everything else – assign **Work: High**
```

Comment:
Mention the `Work` label at the beginning of the point and then explain when to assign it.

Reply:
Done